src/tools/perf/lib/libperf.c

```diff
@@ -1186,22 +1212,37 @@ static ucs_status_t ucp_perf_test_setup_endpoints(ucx_perf_context_t *perf,
                                                   uint64_t features)
 {
     ucs_status_t status;
+    unsigned group_index;
+    unsigned group_size;
+    unsigned peer_index;
     unsigned i;
 
+    group_size  = rte_call(perf, group_size);
+
+    if ((group_size != 2) &&
+            !(perf->params.flags & UCX_PERF_TEST_FLAG_LOOPBACK)) {
+        ucs_error("perftest requires group size to be exactly 2 "
+                  "(actual group size: %u)", group_size);
+        return UCS_ERR_UNSUPPORTED;
+    }
+
+    group_index = rte_call(perf, group_index);
+    peer_index  = rte_peer_index(group_size, group_index);
```

Comment:
no need for rte_peer_index() funciton now since it's used only once

Reply:
will be used for UCT as well

Reply:
ok